src/main/java/ch/njol/skript/expressions/ExprBookAuthor.java

```diff
@@ -29,63 +11,55 @@
 
 import org.bukkit.event.Event;
 import org.bukkit.inventory.meta.BookMeta;
-import org.bukkit.inventory.meta.ItemMeta;
-import org.eclipse.jdt.annotation.Nullable;
+import org.jetbrains.annotations.Nullable;
 
 @Name("Book Author")
 @Description("The author of a book.")
-@Examples({"on book sign:",
-			"\tmessage \"Book Title: %author of event-item%\""})
+@Examples({
+	"on book sign:",
+		"\tmessage \"Book Title: %author of event-item%\""
+})
 @Since("2.2-dev31")
 public class ExprBookAuthor extends SimplePropertyExpression<ItemType, String> {
-	
+
 	static {
 		register(ExprBookAuthor.class, String.class, "[book] (author|writer|publisher)", "itemtypes");
 	}
-	
+
 	@Nullable
 	@Override
 	public String convert(ItemType item) {
-		ItemMeta meta = item.getItemMeta();
-		
-		if (meta instanceof BookMeta)
-			return ((BookMeta) meta).getAuthor();
-		
-		return null;
+		return item.getItemMeta() instanceof BookMeta bookMeta ? bookMeta.getAuthor() : null;
 	}
-	
+
 	@Nullable
 	@Override
 	public Class<?>[] acceptChange(ChangeMode mode) {
-		if (mode == ChangeMode.SET || mode == ChangeMode.RESET || mode == ChangeMode.DELETE)
-			return CollectionUtils.array(String.class);
-		return null;
+		return switch (mode) {
+			case SET, RESET, DELETE -> CollectionUtils.array(String.class);
+			default -> null;
+		}
```

Comment:
; is missing here... smh kenzie!
```suggestion
		};
```